Correct a build issue that occurs when python is not found
in the path. This check should only be done if swig-python
option is selected.

Signed-off-by Clayton Shotwell <clshotwe@rockwellcollins.com>

--- a/configure.ac	2015-05-15 12:28:07.566060349 -0500
+++ b/configure.ac	2015-05-28 15:07:25.357072800 -0500
@@ -217,8 +217,6 @@
    do_swigify=yes
 fi
 
-AM_PATH_PYTHON(2.7)
-
 AC_ARG_ENABLE(swig-python,
               AC_HELP_STRING([--enable-swig-python],
                              [build SWIG interfaces for Python]),
@@ -227,6 +225,7 @@
    if test ${do_swigify} = no; then
       AC_PROG_SWIG(2.0.0)
    fi
+   AM_PATH_PYTHON(2.7)
    SWIG_PYTHON
    do_swigify_python=yes
    do_swigify=yes
